### PR TITLE
Fix Lock, Unlock, and Odometer

### PR DESCRIPTION
**Pull request by GregChan**
# Reviewer Checklist:

- [x] Use the SDK with a valid access token and make requests to all endpoints

### CODE REVIEW

File: src/main/java/com/smartcar/sdk/Vehicle.java [hunk 1]
```diff
@@ -162,9 +165,8 @@ public SmartcarResponse location() throws SmartcarException {
    * @throws SmartcarException if the request is unsuccessful
    */
   public void unlock() throws SmartcarException {
-    RequestBody body = new FormBody.Builder()
-      .add("action", "UNLOCK")
-      .build();
+    String json = "{\"action\": \"UNLOCK\"}";
```
Comment (sankethkatta) [inline src/main/java/com/smartcar/sdk/Vehicle.java]: is there a better way to build this without using a string? I think building a `JSONObject` would be better:

```java
JSONObject json = Json.createObjectBuilder()
  .add("action", "UNLOCK")
  .build();
RequestBody body = RequestBody.create(JSON, json.toString());
```

I think that would work, probably have to check if the `.toString()` just works the way we'd want it to.

File: src/main/java/com/smartcar/sdk/Vehicle.java [hunk 2]
```diff
@@ -175,9 +177,8 @@ public void unlock() throws SmartcarException {
    * @throws SmartcarException if the request is unsuccessful
    */
   public void lock() throws SmartcarException {
-    RequestBody body = new FormBody.Builder()
-      .add("action", "LOCK")
-      .build();
+    String json = "{\"action\": \"LOCK\"}";
```
Comment (sankethkatta) [inline src/main/java/com/smartcar/sdk/Vehicle.java]: same here

File: src/main/java/com/smartcar/sdk/Vehicle.java (hunk 1 above)
Comment (GregChan) [reply src/main/java/com/smartcar/sdk/Vehicle.java]: Wan't aware this existed. I'll change it for lock and unlock.

File: src/main/java/com/smartcar/sdk/Vehicle.java [hunk 3]
```diff
@@ -2,15 +2,21 @@
...
 import okhttp3.HttpUrl;
 import okhttp3.Request;
 import okhttp3.RequestBody;
+import okhttp3.MediaType;
 
 /**
  * Smartcar Vehicle API Object
  */
 public class Vehicle extends ApiClient {
+  public static final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
```
Comment (GregChan) [inline src/main/java/com/smartcar/sdk/Vehicle.java]: Not sure if this should be in `Vehicle.java` or `ApiClient.java` or if there's a constants file that would be better for this.

Comment (RogWilco) [reply src/main/java/com/smartcar/sdk/Vehicle.java]: `ApiClient.java` would probably make the most sense.

File: src/test/java/com/smartcar/sdk/VehicleTest.java [hunk 4]
```diff
@@ -131,9 +133,10 @@ public void testUnlock() throws Exception {
 
     ApiData data = new ApiData();
     SmartcarResponse res = new SmartcarResponse(data);
-    RequestBody body = new FormBody.Builder()
-      .add("action", "UNLOCK")
+    JsonObject json = Json.createObjectBuilder()
+      .add("action", "LOCK")
```
Comment (jerzzhang) [inline src/test/java/com/smartcar/sdk/VehicleTest.java]: Why is this LOCK now?

File: src/test/java/com/smartcar/sdk/VehicleTest.java [hunk 5]
```diff
@@ -146,9 +149,11 @@ public void testLock() throws Exception {
 
     ApiData data = new ApiData();
     SmartcarResponse res = new SmartcarResponse(data);
-    RequestBody body = new FormBody.Builder()
-      .add("action", "LOCK")
+    JsonObject json = Json.createObjectBuilder()
+      .add("action", "UNLOCK")
```
Comment (jerzzhang) [inline src/test/java/com/smartcar/sdk/VehicleTest.java]: Why is this UNLOCK now?

Comment (GregChan) [reply src/test/java/com/smartcar/sdk/VehicleTest.java]: Good catch! Hopefully integration tests will catch mistakes like this.

**Pull request merged**